airbyte-webapp/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,234 @@
+import classNames from "classnames";
+import { Line } from "rc-progress";
+import { useState } from "react";
+import { useIntl } from "react-intl";
+
+import { getJobStatus } from "components/JobItem/JobItem";
+
+import { AttemptRead, JobConfigType, SynchronousJobRead } from "core/request/AirbyteClient";
+import Status from "core/statuses";
+import { JobsWithJobs } from "pages/ConnectionPage/pages/ConnectionItemPage/JobsList";
+import { formatBytes } from "utils/numberHelper";
+
+import styles from "./ProgressBar.module.scss";
+
+function isJobsWithJobs(job: JobsWithJobs | SynchronousJobRead): job is JobsWithJobs {
+  return (job as JobsWithJobs).attempts !== undefined;
+}
+
+const buttonUnStyle = {
```

Comment:
We should never manage styles like this in TypeScript. Those should be going into a SASS module.